constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java

```diff
@@ -502,7 +502,7 @@ boolean isEnabled() {
         return !mDisabled;
     }
 
-    void setEnable(boolean enable) {
+    void setEnabled(boolean enable) {
```

Comment:
This is part of the public API, but only recently introduced in the `2.1.0-alphaX` work - @camaelon @jafu888  what do you think?

Reply:
I don't think it's part of public API - method is package private.

Reply:
Change it